docs/getting_started.md

```diff
@@ -1,64 +1,53 @@
-# Get started
+# Getting Started with Streamlit
 
-The easiest way to learn how to use Streamlit is to try things out yourself. As
-you read through this guide, test each method. As long as your app is running,
-every time you add a new element to your script and save, Streamlit's UI will
-ask if you'd like to rerun the app and view the changes. This allows you to
-work in a fast interactive loop: you write some code, save it, review the
-output, write some more, and so on, until you're happy with the results. The
-goal is to use Streamlit to create an interactive app for your data or model
-and along the way to use Streamlit to review, debug, perfect, and share your
-code.
+The easiest way to learn Streamlit is to try things out yourself! As
+you work through this getting started guide, be sure to run each line of code.
+Each time you make a change to your script and save, the Streamlit UI will
+ask if you'd like to rerun the app and view the changes. This workflow allows for
+working in a fast, interactive loop: write some code, save it, review the
+output, write some more, and so on until you're happy with the results.
 
-Use the links below to jump to a specific section:
+## Part 1: Installing Streamlit in a virtual environment
 
-```eval_rst
-.. contents::
-    :local:
-    :depth: 1
-```
-
-## Prerequisites
+Regardless of which package manager you choose ([pipenv](https://pipenv.pypa.io/en/latest/),
+[conda](https://www.anaconda.com/distribution/), [venv](https://docs.python.org/3/library/venv.html), etc.),
```

Comment:
This is a little misleading, as Streamlit is not actually available with the `conda` package manager. The thing is `conda` is many things: a package manager, a virtual environment manager, a kitchen sink. If you use `conda` for virtual environments, that's fine: but you will need to install Streamlit with `pip`. (BTW, we could add Streamlit as a conda package too, but would take a little work)

Also, `venv` is not a package manager :wink:

Reply:
Yes, it should really be "environment manager".